### DISCUSSION TITLE
Use rust-toolchain while outside of project directory?

### DISCUSSION THREAD

**Discussion by Klaaktu** [shadowsocks/shadowsocks-rust#628] [Q&A] [answered]
_Sorry for the off-topic, but..._

When I try: 
`RUSTFLAGS="-C target-cpu=native" cargo build --release --bin ssserver --manifest-path ~/shadowsocks-rust-master/Cargo.toml`
It says:
```error[E0554]: `#![feature]` may not be used on the stable release channel```

`cd` into the directory works fine. Is there a way to let cargo use `rust-toolchain` while outside of project directory?

**Comment by kallydev** [comment, marked as answer]
Just use one line command to download specified version and set it as default.

```bash
rustup default nightly-2021-09-10
```
